Common/src/main/resources/assets/biomeswevegone/lang/ru_ru.json

```
@@ -123,21 +136,21 @@
   "block.biomeswevegone.beach_grass": "Пляжная трава",
   "block.biomeswevegone.begonia": "Бегония",
   "block.biomeswevegone.bistort": "Змеевик",
-  "block.biomeswevegone.black_ice": "Черный лед",
-  "block.biomeswevegone.black_rose": "Черная роза",
-  "block.biomeswevegone.black_sand": "Черный песок",
-  "block.biomeswevegone.black_sandstone_slab": "Плита из черного песчаника",
-  "block.biomeswevegone.black_sandstone_stairs": "Ступени из черного песчаника",
-  "block.biomeswevegone.black_sandstone_wall": "Стена из черного песчаника",
-  "block.biomeswevegone.black_sandstone": "Черный песчаник",
+  "block.biomeswevegone.black_ice": "Чёрный лед",
+  "block.biomeswevegone.black_rose": "Чёрная роза",
+  "block.biomeswevegone.black_sand": "Чёрный песок",
+  "block.biomeswevegone.black_sandstone_slab": "Плита из чёрного песчаника",
+  "block.biomeswevegone.black_sandstone_stairs": "Ступени из чёрного песчаника",
+  "block.biomeswevegone.black_sandstone_wall": "Стена из чёрного песчаника",
```

По офф. переводу все "стены" должны стать "оградами"
```suggestion
  "block.biomeswevegone.black_sandstone_wall": "Ограда из чёрного песчаника",
```